<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMetaInfo</class>
 <widget class="QWidget" name="QtMetaInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMetaInfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labBoy">
       <property name="text">
        <string>男</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoy">
       <property name="value">
        <number>17</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnBoyInc">
       <property name="text">
        <string>Boy+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnQtMeta">
       <property name="text">
        <string>QtMetaInfo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labGirl">
       <property name="text">
        <string>女</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinGirl">
       <property name="value">
        <number>17</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnGirlInc">
       <property name="text">
        <string>Girl+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtPrint"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
